config/index.js

```diff
@@ -23,8 +23,8 @@ var config = {};
 // validate configuration
 var params = [
   'IBERA_SERVICES_ENDPOINT',
-  'DOCUMENT_SERVICES_ENDPOINT',
-  'OUTLOOK_SERVICE_ENDPOINT'
+  'OUTLOOK_SERVICE_ENDPOINT',
+  'AZURE_OI_STORAGE_CONNECTION_STRING'
```

Comment:
OI doesn't say much... please change to
AZURE_OUTLOOK_STORAGE_CONNECTION_STRING or just STORAGE_CONNECTION_STRING.
this is a different service, there's no problem calling this just STORAGE_CONNECTION_STRING

Reply:
Changed (Also in the automation PS script)